main.go

```diff
@@ -291,11 +293,19 @@ func main() {
 	}
 
 	// Start background workers
-	worker := worker.NewTokenCleanupWorker(context.Background(), appDB)
-	// Activate token cleanup once every hour
-	worker.Start(time.NewTicker(time.Hour))
-	// Stop the worker when the app shuts down
-	defer worker.Stop()
+	// token cleanup, running once every hour
+	tokenCleanupWorker := tokenworker.NewTokenCleanupWorker(context.Background(), appDB)
+	tokenCleanupWorker.Start(time.Hour)
+	// user deactivation and notification workers, running once per day
+	ctx := manager.ContextWithTokenManager(context.Background(), tokenManager)
+	ctx = context.WithValue(worker.LockOwner, config.GetPodName())
+	userDeactivationWorker := userworker.NewUserDeactivationWorker(ctx, appDB)
+	userDeactivationWorker.Start(time.Hour * 24)
```

Comment:
The `defaultUserDeactivationFetchLimit` is set to 10. It means that we will deactivate 10 users every 24 hours. It's not even nearly enough :)

We don't have any delays between deactivation calls for particular users within the cycle, do we?
If so then we don't really want to raise the defaultUserDeactivationFetchLimit to some very big number since it may overload the other services.
So, let's increase the cycle frequency instead then. Let's say 1 minute. So, we will be deactivating up to 10 users per minute or 14400 per 24 hours. It should be good enough.

Also let's put this to configuration. We don't need any config maps for that yet. It's just matter of having all these numbers in one place (default config) so it's easy to track/change them. With some good comment, so, it's clear how these numbers affects the cycle.

Reply:
@alexeykazakov those are default settings, to avoid running on the whole set of users unless explicitly configured. We'll increase the limit and frequency once we're confident with the whole workflow, of course ;)
We already have a "wait period" between each deactivation: https://github.com/fabric8-services/fabric8-auth/blob/master/authentication/account/service/user.go#L154-L155

But I'm fine with increasing the frequency to 1min by default

Reply:
@alexeykazakov timer set to 1min for both workers in 80d413e

Reply:
> We already have a "wait period" between each deactivation: https://github.com/fabric8-services/fabric8-auth/blob/master/authentication/account/service/user.go#L154-L155

This is for user notifications. Not deactivation. But that's fine.

BTW there is some inconsistency between deactivation and notification workers and services. For notifications we iterate users in the user service https://github.com/fabric8-services/fabric8-auth/blob/master/authentication/account/service/user.go#L122 And in the worker just calls that service.
But for deactivation we iterate users in the worker (not in the service). And in the service we have two functions only: Deactivate() a user and GetList().

It would be more consistent if ether:
Move NotifyIdentitiesBeforeDeactivation() to a worker in keep Notify() and GetListToNotify() in the service.
OR
Create DeactivateUsers() in the user service and use it from the worker.

But it's not critical :) I'm OK to keep it as it is.

Reply:
ok, let me refactor this code. The aforementioned `wait period` was supposed to be used for the deactivations, not the notifications.
I suggest I address those changes in another PR to keep this one focused on the workers.